libs/m4i-data-management/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "m4i-data-management"
+version = "1.0.0"
+description = "Library of Data Management"
+authors = [ "Aurelius Enterprise <info@aureliusenterprise.com>" ]
+license = "UNLICENSED"
+readme = "README.md"
+
+  [[tool.poetry.packages]]
+  include = "m4i_data_management"
+
+    [tool.poetry.dependencies]
+    python = "~3.8"
+    dataclasses-json = "~0.6"
+    dict_hash = "~1.3"
```

Comment:
Can you please make sure that all external dependencies used by this library are listed here? For example I saw pandas being imported in some of the modules.